joomla/joomla-cms
# [4.0] Implement the workflow into the multilang sample data

Pull Request for Issue #21707 . (second part)

### Summary of Changes
This PR let the multilang sample data creating a new Workflow to ensure, that the articles are published

### Testing Instructions
- Install a 2nd language
- Apply the "Multilingual Sample Data" (from the dashboard)
- check the workflows, a new workflow called "Sample Workflow (xx-xx)" is created and assigned to the specific new sample data category (xx-xx is the language tag).
